<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemBorderLabel</class>
 <widget class="QWidget" name="ItemBorderLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#wBg{background-color: rgba(255, 255, 255, 0);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="wBorder01" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 25, 55);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="wBorder02" native="true">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/border/res/images/border/border1_02.png);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="wBorder03" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 25, 55);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="wBorder04" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="wBorder05" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#wBorder05{background-color: rgba(255, 255, 255, 0);}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="wBorder06" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="wBorder07" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 25, 55);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="wBorder08" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QWidget" name="wBorder09" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 25, 55);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
